src/modules/flow/utils/domParserUtils.ts

```
@@ -0,0 +1,44 @@
+      "DOMParser polyfill not available. Please install @xmldom/xmldom package."
+    );
+  }
+} else {
+  // In browser, use the native implementation
+  DOMParserImpl = window.DOMParser;
+}
+
+/**
+ * Create a DOMParser instance that works in both browser and Node.js
+ */
+export function createDOMParser(): any {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)